### ISSUE TITLE
[BUG] config parsing fails on initial run

### ISSUE THREAD

**Issue by guseggert** [dlvhdr/gh-dash#478]
**Describe the bug**
I installed gh-dash with `gh extension install dlvhdr/gh-dash` and ran `gh dash`, it crashes due to config parsing errors:

```shell
2024-11-09T10:32:38-05:00 FATA <ui/ui.go:107> Reading config file: failed parsing config file location=""
                           err=
                                 │ failed parsing config.yml: Key: 'Config.theme.colors.Inline.text.secondary' Error:Field validation for 'secondary' failed on the 'hexcolor' tag
                    │ Key: 'Config.theme.colors.Inline.text.inverted' Error:Field validation for 'inverted' failed on the 'hexcolor' tag
                                                          │ Key: 'Config.theme.colors.Inline.text.faint' Error:Field validation for 'faint' failed on the 'hexcolor' tag
          │ Key: 'Config.theme.colors.Inline.text.warning' Error:Field validation for 'warning' failed on the 'hexcolor' tag
                                              │ Key: 'Config.theme.colors.Inline.text.success' Error:Field validation for 'success' failed on the 'hexcolor' tag
  │ Key: 'Config.theme.colors.Inline.background.selected' Error:Field validation for 'selected' failed on the 'hexcolor' tag
                                              │ Key: 'Config.theme.colors.Inline.border.primary' Error:Field validation for 'primary' failed on the 'hexcolor' tag
    │ Key: 'Config.theme.colors.Inline.border.secondary' Error:Field validation for 'secondary' failed on the 'hexcolor' tag
                                              │ Key: 'Config.theme.colors.Inline.border.faint' Error:Field validation for 'faint' failed on the 'hexcolor' tag

```

And also leaves my terminal in a messed up state (no text cursor, newlines messed up, etc.).

**To Reproduce**
1. `gh extension install dlvhdr/gh-dash`
2. `gh dash`

**Expected behavior**
Not sure, I wanted to try it out, I'd imagine crashing is not the expected behavior for new users though.

**Screenshots**
n/a

**Desktop (please complete the following information):**
 - OS: MacOS 15.0.1 (Sequoia)
 - I also tried this in an Ubuntu docker image, and on a Linux laptop (bluefin), same error
 
```shell
$ gh --version
gh version 2.59.0 (2024-10-15)
```

**Comment by cosgroveb** [comment]
I'm having the same issue

**Comment by cosgroveb** [comment]
> Not sure, I wanted to try it out, I'd imagine crashing is not the expected behavior for new users though.

@guseggert it's a pretty cool tool, to be honest. Seems like the bleeding edge releases sometimes have crashes unfortunately. Right now, for me `gh extension install dlvhdr/gh-dash --pin v4.5.4` is working well and is a fairly recent release. Good luck.

**Comment by dlvhdr** [comment]
Hmm yeah it's probably because I don't have really any tests, given it's a side project. I should add some though, to prevent this. Anyway, will look into it.

**Comment by dlvhdr** [comment]
Should be fixed by #480. Please verify.

**Comment by cosgroveb** [comment]
> Hmm yeah it's probably because I don't have really any tests, given it's a side project. I should add some though, to prevent this. Anyway, will look into it. I suspect it happens when you don't have a config

Hey no worries didn't mean my comment as any kind of dig. Really enjoying gh dash

**Comment by guseggert** [comment]
> Should be fixed by #480. Please verify once release is live.

Works great now, thanks!
